stock_move_actual_date/hooks.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2024 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
```

Comment:
```suggestion
from odoo.tools.sql import column_exists
```
We don't need openupgradelib?

Reply:
Thanks! It is valid.